volume/local/local_unix.go

```diff
@@ -71,6 +68,44 @@ func (v *localVolume) mount() error {
 	if v.opts.MountDevice == "" {
 		return fmt.Errorf("missing device in volume options")
 	}
+	if strings.Contains(v.opts.MountOpts, "addr=") && v.opts.MountOpts != "" && v.opts.MountType != "" {
+		addrValue := GetAddress(v.opts.MountOpts)
+		if addrValue != "" {
+			if !IsIP(addrValue) {
```

Comment:
No need for `IsIP`. `net.ResolveIPAddr` will work just fine here if it is a hostname or an IP.